kyuubi-hive-jdbc/src/main/java/org/apache/kyuubi/jdbc/hive/KyuubiStatement.java

```diff
@@ -512,6 +512,36 @@ public ResultSet executeScala(String code) throws SQLException {
     return resultSet;
   }
 
+  public void executeSetCurrentCatalog(String catalog) throws SQLException {
+    if (executeWithConfOverlay(
+        "", Collections.singletonMap("kyuubi.operation.set.current.catalog", catalog))) {
+      resultSet.close();
+    }
+  }
+
+  public ResultSet executeGetCurrentCatalog() throws SQLException {
+    if (!executeWithConfOverlay(
+        "", Collections.singletonMap("kyuubi.operation.get.current.catalog", ""))) {
```

Comment:
any special reason to use such hacky way? no SQL supported?

Reply:
Explained in https://github.com/apache/incubator-kyuubi/issues/2721

Reply:
Because different engines get and set the sql of the catalog database differently

|       | setCatalog            | getCatalog                            | setSchema | getSchema                 |
| ----- | --------------------- | ------------------------------------- | --------- | ------------------------- |
| Spark | USE catalog_1.db_1    | SELECT current_catalog()  (Since 3.1) | USE db_1  | SELECT current_database() |
| Flink | USE CATALOG catalog_1 | SHOW CURRENT CATALOG                  | USE db_1  | SHOW CURRENT DATABASE     |
| Trino | USE catalog_1.db_1    | x                                     | USE db_1  | x                         |
| Hive  | x                     | x                                     | USE db_1  | SELECT current_database() |

Reply:
How about using placeholders for the `statement` argement?

Reply:
For example `SELECT 'kyuubi.operation.get.current.catalog'` ?

Reply:
yes, maybe something like `_SET_CATALOG_${catalogName} / _GET_CATALOG`